spec/listing/listing_spec.js

```diff
@@ -1,97 +1,88 @@
-import { default as Backdraft } from "../../../src/entry.js";
+import List from "../../src/listing/list";
+import Item from "../../src/listing/item";
 
-describe("Listing Plugin", function() {
+import Collection from "../../src/collection";
 
-  var app;
-  var baseExports;
-  var collection;
+import { default as Backdraft } from "../../src/entry";
+
+class Sub extends Item {
+  get tagName() { return "li"; }
+
+  render() {
+    this.$el.html(this.model.get("name"));
+    return this;
+  }
+}
+
+class Main extends List {
+  get tagName() { return "ul"; }
+  get itemClass() { return Sub; }
+}
+
+describe("Listing", function() {
+  let collection;
 
   beforeEach(function() {
-    Backdraft.app.destroyAll();
-    app = Backdraft.app("myapp", {
-      plugins: [ "Listing" ]
-    });
-    app.model("M", {});
-    app.collection("col", {
-      model: app.Models.M
-    });
-    collection = new app.Collections.col();
-    baseExports = Backdraft.plugin("Base");
+    collection = new Collection();
   });
 
   describe("exports", function() {
-    var Listing;
-
-    beforeEach(function() {
-      Listing = Backdraft.plugin("Listing");
+    it("internal classes are available on Backdraft.Listing", function() {
```

Comment:
Grammar doesn't make sense here. Maybe rename to `it("should make internal classes available on Backdraft.Listing", function() {});`

Reply:
Yeah. I was thinking `it` might read as "its".